pyproject.toml

```diff
@@ -28,7 +28,7 @@ requires-python = ">=3.9"
 
 dependencies = [
     "tree-sitter==0.23.2",
-    "tree-sitter-zeek==0.1.1",
+    "tree-sitter-zeek @ git+https://github.com/zeek/tree-sitter-zeek.git",
```

Comment:
We should use a tagged release of tree-sitter-zeek which we currently do not have.